### ISSUE TITLE
 (Bug report)

### ISSUE THREAD

**Issue by dagger-palace** [zymex22/Project-RimFactory-Revived#537]
Hi. I am using Project RimFactory Revived for the first time, and I am encountering strange behavior. I am reluctant to say that this is a bug because I am completely new to the mod, so maybe I am doing something wrong, or maybe I don't understand what is supposed to happen. Nevertheless, I'm seeing what seems to be buggy behavior.

So far, I've built three buildings: Industrial Furnace, Industrial Automatic Mason, and Meat grinder.
- The Industrial Furnace recognizes when a steel slag chunk is in its input zone, says "Smelt metal from slag", and outputs steel to its output square, but it never consumes the steel slag chunk. It is effectively producing an endless supply of steel for free.
- The Industrial Automatic Mason recognizes when a marble chunk is in its input zone, says "Make marble blocks", and outputs marble blocks to its output square, but it never consumes the marble chunk. It is effectively producing an endless supply of marble blocks for free.
- The Meat grinder recognizes when a corpse is in its input zone and says "Butcher creature", but it never consumes the corpse, and it never outputs anything to its output square. It is effectively doing nothing.

I've searched this forum and the discussion forum and comments on Steam, but I haven't seen anyone describe a problem quite like this.

I am using the latest version of the mod -- the one updated Oct 24 @ 10:57.

I am not using either of the two mods listed as incompatible: Mend and Recycle, and Technology Blueprints & Reverse Engineering.

I believe I've changed no mods mid-game, but I'm not sure about that.

-Bridget

**Comment by Sn1p3rr3c0n** [comment]
@dagger-palace 
Please attach a Hugs Lib Log.
1. Observe the issue in game
2. CTRL & F12
3. Post that link here

**Comment by dagger-palace** [comment]
Here you go:
https://gist.github.com/19218043f380675b070b35f46fc2237f

**Comment by Sn1p3rr3c0n** [comment]
@dagger-palace
can you check if you can reproduce this issue:
1. on a new save
2. Without CE (on a new save)

**Regarding the logfile**

The only some what related thing is:
```
S.A.L.: Had no thingRecord of items being accepted, but crafting recipe has ingredients. Did you reload a save? 
```
that seems a bit odd to me that this would be logged (first time seeing that).

more noticeably there are no mentions of the 3 buildings you mentioned issues with: `Industrial Furnace` , `Industrial Automatic Mason` and `Meat grinder`.
Did you have theme placed and behaving faulty before uploading the Hugs Lib Log? (in the same session)

**Comment by dagger-palace** [comment]
> can you check if you can reproduce this issue: 1. on a new save; 2. Without CE (on a new save)

Yes. I created a new game using only Harmony, HugsLib, and Project RimFactory Revived. I turned on Development Mode and created an Industrial Automatic Mason, then I turned off Development Mode and observed the behavior. The Industrial Automatic Mason behaves the same as before: I get an endless supply of stone blocks without any stone chunk ever being consumed. The log is here: https://gist.github.com/6c6484ef853bceabe72c55c5ac6bb53b

> Did you have theme placed and behaving faulty before uploading the Hugs Lib Log? (in the same session)

Yes. In this case, I let the mason run through two cycles of stone creation – and fail to consume two chunks – before uploading the HugsLib log.
